.travis.yml

```diff
@@ -21,10 +21,12 @@ before_install:
   - source activate test-environment
   - conda install -c conda-forge pytest pytest-cov nodejs pscript
   - pip install coveralls
+  - pip install flake8
 install:
   - pip install -e .
 script:
+  - flake8 ipysheet
```

Comment:
will travis stop directly if flake fails or will it still run the pytest?

Reply:
Good question, I'll trigger an invalid build just to see if that stops everything

Reply:
Yes, the build fails when flake8 doesn't pass 
![flake8](https://user-images.githubusercontent.com/21197331/52349015-e7a4bb00-2a25-11e9-946e-d7b723440852.png)

Reply:
But I guess it's fine, it could be convenient when flake8 is not passing but we still want to know if tests are passing, don't you think?

Reply:
Exactly, just what I hoped for, and consistent with the docs:
https://docs.travis-ci.com/user/job-lifecycle/